main-node/repeater/default_repeater.py

```diff
@@ -15,6 +14,7 @@ def decision_function(self, history, point, **configuration):
         In other case - compute result as average between all experiments.
         :param history: history class object that stores all experiments results
         :param point: concrete experiment configuration that is evaluating
+                      shape - tuple, e.g. ``(1200, 32)``
```

Comment:
mainly it was a list.
In case of retrieving the point from the regression model it was a tuple. (and caused a bug.)
Fixed in current [MKDE branch](https://github.com/Valavanca/benchmark/commit/e645ae4c196d1d8de1f1e4213c765a1e4d84ebec#diff-c5cc9d359a373d7664f836cdfe29612bR28) to return a list